Kernel/System/Ticket/Article.pm

```diff
@@ -1197,17 +1197,21 @@ sub _MetaArticleList {
         return;
     }
 
-    my $CacheKey = '_MetaArticleList::' . $Param{TicketID};
+    my $CacheKey
```

Comment:
Just a bit nitpicking. For VersionView and ShowDeletedArticles any true value could be passed. But the later cache clearing code only clears for the values 0 and 1. This edge case could be avoided by using dedicated variables 
`my $VersionView = $Param{VersionView} ? 1 : 0;`

Reply:
If hypothetically `VersionView => 99.9` is passed, than int('99.9') = 99 would be used as Key for getting and setting the cache. But 1 would still be the Key for clearing the cache.

Reply:
I see - you're right, as usual ;) I'll change this